utilities/src/main/java/io/syndesis/qe/utils/TestUtils.java

```diff
@@ -158,4 +161,37 @@ public static void sleepIgnoreInterrupt(long milis) {
             e.printStackTrace();
         }
     }
+
+    public static void updateCredentialJson(String connectionName, DBAllocation dbAllocation) {
+
+        Map<String, String> allocProperties = dbAllocation.getAllocationMap();
+
+        Optional<Account> optional = AccountsDirectory.getInstance().getAccount(connectionName);
+
+        if (optional.isPresent()) {
+            TestUtils.resetDBProperties(optional.get(), allocProperties);
+        } else {
+            throw new RuntimeException("There is no " + connectionName + " input in your credential json!");
+        }
+    }
+
+    //        url -> db.jdbc_url, user -> db.username, password -> db.password, schema -> db.name
+    private static void resetDBProperties(Account account, Map<String, String> allocMap) {
+        Map<String, String> properties = account.getProperties();
+        if (properties == null) {
+            account.setProperties(new HashMap<>());
+            properties = account.getProperties();
+        }
+        switch (account.getService()) {
+            case "oracle12":
```

Comment:
If name of credentials are the same for all dbs, switch is not needed.

Reply:
I made this method more universal, (setAccountProperties) so I keep the switch.